### PR TITLE
New pull request on click of "Prepare Release"

### PR DESCRIPTION
**Pull request by abinbaby-sap**

### CODE REVIEW

File: src/zcl_hello_world.clas.abap [hunk 1]
```diff
@@ -13,7 +13,9 @@ ENDCLASS.
 CLASS zcl_hello_world IMPLEMENTATION.
 
   METHOD if_oo_adt_classrun~main.
-    data(message) = |hello world|.
+    data(name) = out->get( ).
+    data(message) = |hello { name }|.
```
Comment (sunilb) [inline src/zcl_hello_world.clas.abap]: Please do I18n of this string.

**Review by sunilb — COMMENTED** [review]
You can consider providing I18n of this string.

File: src/zcl_hello_world.clas.abap (hunk 1 above)
Comment (abinbaby-sap) [reply src/zcl_hello_world.clas.abap]: resolved

Comment (sunilb) [reply src/zcl_hello_world.clas.abap]: Cool! Fine with me.

**Review by sunilb — CHANGES_REQUESTED** [review]
Please work on the modularization of the code and revert.

**Comment by abinbaby-sap** [comment]
> Please work on the modularization of the code and revert.

changes are incorporated

**Review by sunilb — APPROVED** [review]

**Pull request merged**